tests/test-cases/Group23-VIC-Machine-Service/23-06-VCH-Certificate.robot

```diff
@@ -0,0 +1,95 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 23-06 - VCH Certificate
+Resource  ../../resources/Util.robot
+Suite Setup  Start VIC Machine Server
+Suite Teardown  Terminate All Processes  kill=True
+Test Teardown  Cleanup VIC Appliance On Test Server
+Default Tags
+
+*** Keywords ***
+Install VIC Machine Without TLS
```

Comment:
Is the only difference here from our existing keyword?:
Set Suite Variable  ${vicmachinetls}  --no-tls

Then we really should just add a parameter to our existing keyword...